File: Data Structures & Algorithms/find-target-in-rotated-sorted-array/submission-0.java
class Solution {
    public int search(int[] nums, int target) {
        int n=nums.length;
        int[] num2=new int[n];
        for(int i=0;i<n;i++){
            num2[i]=nums[i];
        }
        Arrays.sort(nums);
        for(int i=0;i<n;i++){
            if(target==num2[i]){
                return i;
            }

        }
        return -1;
        
    }
}

File: Data Structures & Algorithms/combination-target-sum/submission-0.py
class Solution:
    def combinationSum(self, nums: List[int], target: int) -> List[List[int]]:
        sol=[]
        res=[]
        n=len(nums)
        def backtrack(i,total):
            if total==target:
                res.append(sol[:])
                return
            if total>target or i==n:
                return
            backtrack(i+1,total)
            sol.append(nums[i])
            backtrack(i,total+nums[i])
            sol.pop()
        backtrack(0,0)
        return res
